app/plugins/views.js

```diff
@@ -33,11 +33,12 @@ module.exports = {
     relativeTo: __dirname,
     isCached: !config.isDev,
     context: {
+      analyticsAccount: analyticsAccount,
```

Comment:
This value comes from the config file but there is nothing in the config that supports that and we're not using any analytics.  Suggest removal.

Reply:
Thanks, I've removed analyticsAccount from the code